code/modules/bitrunning/objects/gimmick_disks/_gimmick_disk.dm

```diff
@@ -0,0 +1,112 @@
+
+/**
+ * Bitrunning tech disks which let you load full loadouts into the vdom on first avatar generation.
+ */
+/obj/item/bitrunning_disk/gimmick
+	desc = "A disk containing source code. It can be used to preload gimmick loadouts into the virtual domain."
+	/// The selected loadout that this grants
+	var/datum/bitrunning_gimmick/granted_loadout
```

Comment:
is this ref cleared? does it need to be?

Reply:
...good catch, thanks